partner_contact_nutrition_exclusion/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Ursa Information Systems <http://ursainfosystems.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```

Comment:
Use https on license lines (and preferably on copyright as well, unless website really only reachable by http:).